data/data.json

```diff
@@ -1138,10 +1138,10 @@
   },
   "v1.12.9-rancher1-1": {
    "etcd": "rancher/coreos-etcd:v3.2.24-rancher1",
-   "alpine": "rancher/rke-tools:v0.1.34",
-   "nginxProxy": "rancher/rke-tools:v0.1.34",
-   "certDownloader": "rancher/rke-tools:v0.1.34",
-   "kubernetesServicesSidecar": "rancher/rke-tools:v0.1.34",
+   "alpine": "melsayed/rke-tools:v0.1.35",
```

Comment:
this seems by mistake? rancher/rke-tools?

Reply:
n00b mistake. Forgot to recompile after removing my test image.